book/website/communication/open/education.md

```diff
@@ -61,9 +61,12 @@ Fully open OERs comply with the [5 Rs](http://opencontent.org/definition/):
 ## How to set up OERs? 
 
 1. Choose a platform.
-2. Collect resources under an open license (see [Attribution & Fair Use: Copyright in Open Education](https://www.youtube.com/watch?v=jGTUHdadqJU) and {ref}`Licensing data section<rr-licensing-data>` for more information).
-3. Ask for feedback.
-4. Share and promote your resources.
+2. Collect resources under an open license
+    - see [Attribution & Fair Use: Copyright in Open Education](https://www.youtube.com/watch?v=jGTUHdadqJU),
+    - [Copyright ownership in higher education and potential implications for Open Education](https://doi.org/10.17161/jcel.v5i1.14946) by {cite:ps}`Gumb2022oer`,
+    - and {ref}`Licensing data section<rr-licensing-data>` for more information.
```

Comment:
Would it be a better structure to write:

```
2. Collect resources under an open license.
   For more information, see:
    - [Attribution & Fair Use: Copyright in Open Education](https://www.youtube.com/watch?v=jGTUHdadqJU),
    - [Copyright ownership in higher education and potential implications for Open Education](https://doi.org/10.17161/jcel.v5i1.14946) by {cite:ps}`Gumb2022oer`,
    - {ref}`Licensing data section<rr-licensing-data>`.
```

Reply:
Yes, good point - thank you!